graphistry/validate_encodings.py

```diff
@@ -309,7 +309,7 @@ def validate_complex_encoding_badge(kind, mode, name, badge):
 # { 
 # }
 # NOTE: auto-adds graphType, encodingType based on name
-def validate_complex_encoding(kind, mode, name, enc, attributes: list):
+def validate_complex_encoding(kind, mode, name, enc, attributes: list = []):
```

Comment:
@aucahuasi this may be clearer as an `attributes: Optional[List] = None` , as `[]` vs `None` connote different things

Reply:
good catch, done!